styles/layout.css

```diff
@@ -758,20 +758,21 @@ table.dirlist caption {
 #toolbox .char_box {
   padding-top: 1px;
 }
+#toolbox .big_text,
+#toolbox .picker {
+  font-family: DejaVu Sans Mono;
```

Comment:
That's a fair question. My instinct says to keep them as they are since they're more like menus than buttons. Having them in a different font may make it more visually distinct between what you click to get more buttons and what you click to insert a character.

I'm open to discussions on it, however.

Reply:
That's true. I think the main thing that leapt out at me was the cap `O` (cap oh). While some monospaced fonts do make it rounder, neither DVSM nor DPCM2 does, but the font used for the menu buttons does. I, too, would be interested in what others think.

Reply:
@srjfoo and I chatted about this in Slack. We're comfortable moving forward with this as-is, particularly with the knowledge that we can continue to refine these in the future if needed.